dlut-cite-style.csl

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" name-delimiter=", " delimiter-precedes-last="always" initialize-with=" " name-as-sort-order="all" sort-separator=" " names-delimiter=". " demote-non-dropping-particle="never" default-locale="zh-CN">
```

Comment:
You mention this is a multilingual style, but the style has the locale set to chinese. What's the case here?

Reply:
Before I submit the document, I use the method mentioned in [#63](https://github.com/citation-style-language/schema/issues/63) to output Chinese and English citations at the same time. Just because it can output appropriate Chinese and English citations at the same time, I name it multi Lang
Unfortunately, after submitting, I found that this method was not accepted by the current standard

Reply:
I only started working with CSL today and am looking for other ways to support multilingualism

Reply:
What are you trying to achieve?

Reply:
Give me some time, and if I haven't found a suitable way to deal with it tomorrow, I'll delete the PR and will submit it when CSL supports it